lib/engine/game/g_1862_solo/company.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require_relative '../../company'
+
+module Engine
+  module Game
+    module G1862Solo
+      class Company < Engine::Company
+        attr_accessor :ipo_row_index
```

Comment:
Are you subclassing Company here only to add this attribute? If so, it might be better to just add a hash to the 62_solo game class that is indexed by the company.

Reply:
I will change that.